app/views/taxons/sections/_news_and_communications.html.erb

```diff
@@ -1 +1,45 @@
-<%= render partial: 'document_list_with_grid', locals: { documents: section[:documents] } %>
+<%
+  featured_item = section[:documents].shift(section[:promoted_content_count]).first
+  documents = section[:documents].any?
+  featured_item_layout_class = "taxon-page__featured-item--single" if !documents
+%>
+
+<div class="grid-row">
+  <div class="taxon-page__featured-item column-one-third <%= featured_item_layout_class %>">
+    <img class="taxon-page__featured-image" src="<%= featured_item[:image][:url] %>" alt="<%= featured_item[:image][:alt] %>">
+    <div class="taxon-page__featured-text">
+      <a class="taxon-page__featured-link" href="<%= featured_item[:link][:path] %>"><%= featured_item[:link][:text] %></a>
+      <div class="taxon-page__featured-metadata-wrapper">
+        <% featured_item[:metadata].each do |metadata_key, metadata_value| %>
+          <% if metadata_key.to_s.eql?("public_updated_at") %>
+            <time class="taxon-page__featured-metadata" datetime="<%= metadata_value.iso8601 %>">
+              <%= l(metadata_value, format: '%e %B %Y') %>
+            </time>
+          <% else %>
+            <p class="taxon-page__featured-metadata"><%= metadata_value %></p>
+          <% end %>
+        <% end %>
+      </div>
+    </div>
+    <% if !documents %>
```

Comment:
I could be wrong about this, but is this logic overly complicated? Am I right in thinking that this 'see more' link should always appear? Then you wouldn't need to have it twice in the code, just take it out of the if statements and put it at the bottom.

(thinking out loud here) If you did do that, it would mean that the see more link would appear like this:

![screen shot 2018-04-26 at 09 34 16](https://user-images.githubusercontent.com/861310/39294972-03317f84-4935-11e8-921e-73ee6b57887c.png)

...which obviously wouldn't be good. However, you might be able to move the logic to the div class instead: `<div class="<%= "column-two-thirds" if documents %>">`

Sorry, just wondering if you'd thought about this.

Reply:
Nice! You're right in thinking the see more link needs to appear and that I put the logic in place purely due to placement of that link, but... I didn't think of moving the logic to the `div`!

I've managed to get that working with `<div class="<%= documents ? "column-two-thirds" : "taxon-page__featured-see-more" %>">`. Great spot, thanks! 👍💯